crates/astria-core/src/sequencerblock/v1alpha1/block.rs

```diff
@@ -1291,86 +1291,23 @@ impl FilteredSequencerBlockError {
 )]
 pub struct Deposit {
```

Comment:
This is just a stupid container without any invariants.

Made all fields public and removed the getters.